Source/Menu/Notifications/NotificationManager.cs

```diff
@@ -0,0 +1,674 @@
+﻿// COPYRIGHT 2009, 2010, 2011, 2012, 2013, 2014, 2015 by the Open Rails project.
```

Comment:
Probably could update the years here :) I've used a range for newer files as it's simpler.

Reply:
Good idea. I read something a while back on the copyright dates and [this looks interesting](https://liferay.dev/blogs/-/blogs/how-and-why-to-properly-write-copyright-statements-in-your-code)